crates/emitter/src/emitter.rs

```diff
@@ -269,6 +269,40 @@ impl Emitter {
         self.process_token(x, false, Some(i))
     }
 
+    fn identifier_with_prefix_suffix(
+        &mut self,
+        identifier: &Identifier,
+        prefix: &Option<String>,
+        suffix: &Option<String>,
+    ) {
+        if prefix.is_some() || suffix.is_some() {
+            let token =
+                self.append_prefix_suffix_to_token(&identifier.identifier_token, prefix, suffix);
+            self.veryl_token(&token);
+        } else {
+            self.veryl_token(&identifier.identifier_token);
+        }
+    }
+
+    fn append_prefix_suffix_to_token(
```

Comment:
How about changing `append` of `VerylToken` like below?
If so, the `append` can be used instead of this function directly.

```
pub fn append(&self, prefix: Option<&str>, postfix: Option<&str>) -> Self {
```

Reply:
I've changed types of prefix and suffix.